saas_app/submit_request/forms.py

```diff
@@ -30,15 +30,13 @@ def __init__(self, *args, **kwargs):
         self.helper.form_class = "blueForms"
         self.helper.form_method = "post"
         self.helper.form_action = "submit_saas_form"
-        self.helper.add_input(
-            Submit("submit", _("Submit"), css_class="btn btn-primary")
-        )
+        self.helper.add_input(Submit("submit", _("Submit")))
         self.helper.add_input(
             Button(
                 "cancel",
                 _("Cancel"),
                 css_id="submit",
-                css_class="btn btn-primary btn-lg",
+                css_class="btn btn-primary btn-med",
```

Comment:
It's the first time I see a `btn-med` class, is that custom or part of crispy?

Reply:
You know what, you are totally right! In general btn-lg, btn-sm are part of bootstrap but btn-med is not (or at least I could not find it now). Good catch! I will play around with this to actually use a button class that is valid.

Reply:
I removed btn-med (no idea where I got it from) and then I made the buttons have a consistent look. As you pointed out, it meets better accessiblity standards. 

<img width="1150" alt="Screen Shot 2023-04-03 at 5 27 00 PM" src="https://user-images.githubusercontent.com/85905333/229655436-926303af-d9b0-4348-ba41-a652552d95d4.png">